lib/sidekiq/api.rb

```diff
@@ -318,7 +318,7 @@ def display_class
       @klass ||= case klass
                  when /\ASidekiq::Extensions::Delayed/
                    safe_load(args[0], klass) do |target, method, _|
-                     "#{target}.#{method}"
+                     "#{target}##{method}"
```

Comment:
@rmachielse I think this the original code is correct. The delayed extension, `SomeClass.delay.class_method`, always uses a [class method](https://github.com/mperham/sidekiq/wiki/Delayed-extensions#class-methods) notation, which is a `.`. It doesn't work for instances of a class:

```ruby
SomeClass.new.delay.class_method
undefined method `delay' for #<SomeClass:0x007f930d10af98>
```

The `#` symbol is used to indicate it's an instance method, which this is not.
We'll need to parse and store any difference between class and instance methods in the AppSignal gem instead (https://github.com/appsignal/appsignal-ruby/pull/348)